CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 0.23.0 / 2023-02-17
+
+- [FEATURE] Add ability to duplicate panels (#949)
+- [FEATURE] UI - Add dashboard CRUD (#939)
+- [FEATURE] Generic Dialog Components (#948)
+- [FEATURE] Initiate project creation/deletion in the frontend (#916)
+- [ENHANCEMENT] Modify placement of duplicate panels (#954)
+- [ENHANCEMENT] project routing and list view links (#953)
+- [ENHANCEMENT] Add color prop to Buttons (#917)
+- [ENHANCEMENT] Remove unnecessary background color (#915)
+- [BUGFIX] Allow editing of time range using inputs (#963)
+- [BUGFIX] time series migration add unit inside y_axis (#952)
+- [BUGFIX] round small decimal values when abbreviate is set (#946)
+- [BUGFIX] Fix empty dashboard spec that wasn't rejected (#931)
+- [BUGFIX] Fix discard changes confirmation dialog showing up even when there is no change (#929)
+- [BUGFIX] Fix increased metadata.version (#925)
+- [BUGFIX] Fix API error EOF that happens when decoding multiple times the body from a middleware (#923)
+- [BREAKINGCHANGE] Move panel padding to plugins (#942)
+- [BREAKINGCHANGE] Add a consistent JSON tab for every panel plugin (#922)
+- [BREAKINGCHANGE] remove deprecated top-level `unit` in time series panel (#920)
+
```

Comment:
I think we add we removed the support of etcd as the main database

Reply:
Hmm is that merged yet, I've seen this [draft PR](https://github.com/perses/perses/pull/961) but is there another previous one? Also, what exactly will that mean, can we keep the same workflow for local dev using file system / local_db?

Reply:
Oh nvm I see what you mean now in https://github.com/perses/perses/pull/933
I'll add that BREAKINGCHANGE note

Reply:
ah yes sorry, I forgot the PR linked. Cool to see you found it !

Reply:
Breaking change note is added now!